src/turbomind/python/bind.cpp

```diff
@@ -52,6 +52,10 @@ DLDevice getDLDevice(triton::Tensor& tensor)
     return device;
 }
 
+static void deleter(DLManagedTensor* arg) {
+  delete static_cast<triton::Tensor*>(arg->manager_ctx);
```

Comment:
I tried to print the `arg->manager_ctx`. It seems always `nullptr`.

Reply:
Seems a measuring error. I'd better do some further tests.